openfisca_nsw_safeguard/ESS_PDRS_Estimator/parameters/HVAC2/activity_eligibility/HVAC2_activity_eligibility_parameters

```diff
@@ -0,0 +1,179 @@
+    }
+
+
+class HVAC2_Installed_by_qualified_person(Variable):
+    value_type = bool
+    entity = Building
+    default_value = True
+    definition_period = ETERNITY
+    label = 'Has the installation of the end-user equipment been performed or supervised by a suitably licensed person?'
+    metadata = {
+        'display_question': 'Has the installation of the end-user equipment been performed or supervised by a suitably licensed person?'
+    }
+
+
+class HVAC2_Equipment_installed(Variable):
+    value_type = bool
+    entity = Building
+    default_value = True
+    definition_period = ETERNITY
+    label = 'Is the new End-User equipment installed and operational?'
+    metadata = {
+        'display_question' : 'Is the new End-User equipment installed and operational?'
+    }
+
+
+class HVAC2_residential_building(Variable):
+    value_type = bool
+    entity = Building
+    default_value = False
+    definition_period = ETERNITY
+    label = 'Has the new End-User equipment been installed in a residential building?'
+    metadata = {
+        'display_question' : 'Has the new End-User equipment been installed in a residential building?'
+    }
+
+
+class HVAC2_equipment_registered_in_GEMS(Variable):
+    value_type = bool
+    entity = Building
+    definition_period = ETERNITY
+    label = 'Is the new air conditioner recorded in the GEMS registry (as defined within the GEMS Determination 2019)?'
+    metadata = {
+        'display_question' : 'Is the new air conditioner recorded in the GEMS registry (as defined within the GEMS Determination 2019)?'
+    }
+
+
+class HVAC2_new_equipment_cooling_capacity(Variable):
+    value_type = bool
+    entity = Building
+    definition_period = ETERNITY
+    label = 'Does the new air conditioner have a cooling capacity recorded in the GEMS registry?'
+    metadata = {
+        'display_question': 'Does the new air conditioner have a cooling capacity recorded in the GEMS registry?'
+    }
+
+
+class HVAC2_TCPSF_greater_than_minimum(Variable):
+    value_type = bool
+    entity = Building
+    label = 'Is your TCPSF equal to or greater than the Minimum TCPSF for the same Product Type and Cooling Capacity in ESS Table F4.5?'
+    definition_period = ETERNITY
+    metadata = {
+        'display_question' : 'Is your TCPSF equal to or greater than the Minimum TCPSF for the same Product Type and Cooling Capacity in ESS Table F4.5?'
+    }
+
+
+class HVAC2_installed_centralised_system_common_area_BCA_Class2_building(Variable):
+    value_type = bool
+    entity = Building
+    definition_period = ETERNITY
+    label = 'Is the installation in a centralised system or common area in a Class 2 building?'
+    metadata = {
+        'variable-type' : 'conditional',
+        'display_question' : 'Is the installation in a centralised system or common area in a Class 2 building?'
+    }
+
+
+#Need some Alisha attention on this one
+class HVAC2_climate_zone(Variable):
```

Comment:
looks good to me. it should be an enum.

could you remove the inline comments?

Reply:
Done!